snp_psir_bringup/launch/hardware_bringup.launch

```diff
@@ -1,9 +1,24 @@
 <?xml version="1.0"?>
 <launch>
+
   <arg name="sim_robot" default="false"/>
-  <arg name="pipeline" default="command_planner" />
+  <arg name="pipeline" default="command_planner"/>
+  <arg name="robot_name" default="psir"/>
+  <arg name="controller_group" default="arm"/>
```

Comment:
Not sure if we need this arg, since yaml is from `psir_description`

Reply:
I added it to make the next PR easier because the controller group name has changed:
https://github.com/rosin-project/automatica18_scan_and_plan_demo/blob/dd4bf28ce8ee2c30491ef084ea41265666784c29/snp_psir_bringup/launch/hardware_bringup.launch#L7